src/main/java/ch/njol/skript/expressions/ExprArmorSlot.java

```diff
@@ -30,17 +31,22 @@
 import ch.njol.skript.util.slot.EquipmentSlot.EquipSlot;
 import ch.njol.skript.util.slot.Slot;
 import ch.njol.util.Kleenean;
-import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.*;
 import org.bukkit.event.Event;
 import org.bukkit.inventory.EntityEquipment;
 import org.eclipse.jdt.annotation.Nullable;
 
-import java.util.Arrays;
-import java.util.Locale;
+import java.util.*;
 import java.util.stream.Stream;
 
 @Name("Armour Slot")
-@Description("Equipment of living entities, i.e. the boots, leggings, chestplate or helmet.")
+@Description({
+	"Equipment of living entities, i.e. the boots, leggings, chestplate or helmet.",
+	"`Body armor` is a special slot that can only be used for:",
+	"    `Horse`: Horse Armor (Zombie and Skeleton Horse's do not work)",
```

Comment:
you should use \t for indentation but i think an unordered list suits this better

Reply:
I put a comment in Discord, maybe it got buried, you can use HTML tags here so you can do something like
```html
<ul>
  <li>Something</li>
  <li>another bullet point</li>
</ul>
```
(`ul` -> unordered list, `ol` -> ordered list)